modules/administration/pages/tuning-changelogs.adoc

```diff
@@ -0,0 +1,16 @@
+[[tuning-changelogs]]
+= Tuning Changelogs
+
+Some packages have a long list of changelog entries.
+This data is downloaded by default, but is very rarely useful information.
+In order to limit the amount of changelog metadata which is downloaded and to save disk space, you can put a limit on how many entries to keep on disk.
+
+This configuration option is in the [filename]``/etc/rhn/rhn.conf`` configuration file.
+The parameter defaults to [systemitem]``0``, which means unlimited.
+
+----
+java.max_changelog_entries = 0
+----
+
+When you set this parameter, it will come into effect only when packages are synchronized.
```

Comment:
> When you set this parameter, it will come into effect only when packages are synchronized.

When you set this parameter, it will come into effect only for new packages synchronized.